plugins/git/functions

```diff
@@ -41,6 +41,21 @@ git_build_app_repo() {
   fi
 }
 
+suppress_output() {
```

Comment:
Academic argument: it's generic enough that it probably should exist in common. regardless, this sort of thing is an option in the golang `DokkuCmd` struct